### PR TITLE
add icon figures and fixing the link in readme

### PR DESCRIPTION
**Pull request by chiaweh2**
A test of using the binder service is attempted in the config file

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ tags:
 
 execute:
   # To execute notebooks via a Binder instead, replace 'cache' with 'binder'
-  execute_notebooks: cache
+  execute_notebooks: binder
```
Comment (mgrover1) [inline _config.yml]:
```suggestion
  execute_notebooks: "binder"
```

Comment (chiaweh2) [reply _config.yml]: pushed new changes that add the fix

**Comment by mgrover1** [comment]
@chiaweh2 how long does this take to run locally?

**Comment by chiaweh2** [comment]
@mgrover1 when I time it yesterday it is about 10 mins from the GitHub provided server. I am timing it now on the server at NOAA.

**Comment by chiaweh2** [comment]
Is there any place that I need to change the timeout besides in the config file for using the binder?

**Comment by chiaweh2** [comment]
@mgrover1 it takes 9 mins on the NOAA server

**Comment by mgrover1** [comment]
It works!! @chiaweh2

**Comment by chiaweh2** [comment]
Thank you so much for the help! @mgrover1

**Comment by mgrover1** [comment]
Happy to help @chiaweh2 !

**Pull request merged**